LondonBritishLibrary/orient/BLorient8823.xml

```diff
@@ -70,22 +70,22 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                         </msItem>
                         <msItem xml:id="ms_i4">
                             <locus from="95ra" to="103va"/>
-                            <title type="incomplete">Three Compositions for Moral Edification.</title>
+                            <title ref="LIT6934Edification"/>
                                 <msItem xml:id="ms_i4.1">
                                     <locus from="95ra" to="97va"/>
-                                    <title>Composition 1</title>
+                                    <title>part 1</title>
```

Comment:
since you have created a structured work record you can use the ref `LIT6934Edification#part1` here